src/native/macos.rs

```diff
@@ -87,9 +87,22 @@ impl crate::native::NativeDisplay for MacosDisplay {
         }
     }
     fn clipboard_get(&mut self) -> Option<String> {
-        None
+        unsafe {
+            let pasteboard: ObjcId = msg_send![class!(NSPasteboard), generalPasteboard];
+            let content: ObjcId = msg_send![pasteboard, stringForType: NSStringPboardType];
+            let string = nsstring_to_string(content);
+            (!string.is_empty()).then(|| string)
```

Comment:
to be honest I would prefer something like
```
if string.is_empty() {
  return None;
}
string
```

Reply:
I totally agree, I've pushed a fix for this.